libraries/lib-numeric-formats/NumericConverterType.h

```diff
@@ -0,0 +1,18 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/**********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file NumericConverterType.h
+
+ Dmitry Vedenko
+
+ **********************************************************************/
+#pragma once
+
+enum class NumericConverterType
```

Comment:
The commit introducing this enum class fails to build.  It does not yet remove the old enumeration.  One line of the ctor of `NumericConverter` should change to

`   if (type == NumericConverterType::TIME )`